examples/set_effective_context/README.md

```diff
@@ -0,0 +1,8 @@
+## set_effective_context
```

Comment:
I would like to rename this example to `postpone_requests` rather than using the actual hostcall name! 👍

Reply:
Done